### DISCUSSION TITLE
Do we have an option to mention user groups instead of users in slack integrations?

### DISCUSSION THREAD

**Discussion by akhilaananthkumar** [test-results-reporter/reporter#163] [Ideas]
`{
  "name": "mentions",
  "inputs": {
    "users": [
      {
        "name": "Jon",
        "slack_uid": "ABCDEFGH"
      }
    ]
  }
}
`

Instead of using **member id** value in slack_uid  field, is there a way to use **group id**?

Slack group id - https://slack.com/intl/en-au/help/articles/212906697-Create-a-user-group

**Comment by ASaiAnudeep** [comment]
Not yet. created an issue to track the work.

**Comment by leelaprasadv** [comment]
@akhilaananthkumar Changes have been rolled out in [v1.1.6](https://github.com/test-results-reporter/reporter/releases/tag/v1.1.6)

Documentation: https://test-results-reporter.github.io/extensions/mentions.html#mentioning-a-user-group-in-slack

**Reply by akhilaananthkumar** [reply]
Thank you, appreciate the quick response :)
